api/package.json

```diff
@@ -18,18 +18,17 @@
   "dependencies": {
     "async": "^3.2.1",
     "bikram-sambat": "^1.7.0",
-    "bikram-sambat-bootstrap": "^1.5.0",
-    "body-parser": "^1.19.2",
+    "body-parser": "^1.20.2",
     "bowser": "^2.11.0",
     "buffer-shims": "^1.0.0",
     "compression": "^1.7.4",
     "express": "^4.18.2",
-    "google-libphonenumber": "^3.2.31",
+    "google-libphonenumber": "^3.2.33",
     "gsm": "^0.1.4",
     "helmet": "^3.23.3",
     "http-proxy": "^1.18.1",
     "locale": "^0.1.0",
-    "lodash": "4.17.19",
+    "lodash": "^4.17.21",
```

Comment:
I think upgrading this is okay since the [issue](https://docs.communityhealthtoolkit.org/contribute/code/core/update-dependencies/) has been spotted only in the Admin app.

Reply:
Hmm.. I'll consider reverting for consistency. Thanks for pointing it out!